symengine/solve.h

```diff
@@ -46,6 +50,13 @@ RCP<const Set> solve_poly_quartic(const vec_basic &coeffs,
                                   const RCP<const Set> &domain
                                   = universalset());
 
+bool is_a_LinearArgTrigEquation(const Basic &b, const Symbol &x);
+
+RCP<const Set> invertComplex(const RCP<const Basic> &fX,
```

Comment:
Can you add a comments here about what this method does? It'd be great to have it for all the methods.